Java/JDI/jdi-mock-rest/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.epam.jdi</groupId>
+    <artifactId>jdi-mock-rest</artifactId>
+    <version>1.0-SNAPSHOT</version>
```

Comment:
As far as I see most of the modules do not have version (perhaps inherit from parent) - please try without version.
Same about <goupId>.